services/opencollective/opencollective-by-tier.service.js

```diff
@@ -21,12 +21,31 @@ export default class OpencollectiveByTier extends OpencollectiveBase {
   }
 
   async handle({ collective, tierId }) {
-    const result = await this.fetchCollectiveBackersCount(collective, {
-      tierId,
+    const {
+      data: {
+        account: {
+          members: { nodes: members },
+        },
+      },
+    } = await this.fetchCollectiveInfo({
+      collective,
+      accountType: ['INDIVIDUAL', 'ORGANIZATION'],
     })
-    if (result.tier) {
-      if (!result.tier.endsWith('s')) result.tier += 's'
-    } else result.tier = 'new tier'
-    return this.constructor.render(result.backersCount, result.tier)
+
+    if (!members) {
+      return this.constructor.render(0, 'open collective')
+    }
+
+    // FIXME: should use pagination to get all members
```

Comment:
Would it be possible to do this properly if `OpencollectiveByTier` made its own GraphQL query rather than sharing a query with the other services, or does the API not support the relevant params to do the queries we would need here?

Reply:
I don't find an API to get the count by tier.

Reply:
https://graphql-docs-v2.opencollective.com/types/Account

<img width="644" alt="image" src="https://github.com/badges/shields/assets/37948597/d6c31a89-93f2-45a1-a613-75255184b17d">

Reply:
I don't want to fix it now in this PR. Do you think it's better to fallback to old rest api for /tier (i.e., keep it as-is)? But I think sooner or later we need to use new api. Besides, opencollective calls it `legacyId` :)

Reply:
I copied the old JsonBase to `services/opencollective/opencollective-by-tier.service.js` and reverted the changes there.

Reply:
I agree for the moment what you've done makes sense. At least with the Rest API we get a larger page size in a single request so we'll be giving a correct result in more cases.

Lets punt where we go next with this one to another issue. I will write something up.

Reply:
https://github.com/badges/shields/issues/9361